### PR TITLE
CI(eos_cli_config_gen): Restructured the eos_cli_config_gen molecule host vars part-5

### PR DESCRIPTION
**Pull request by MaheshGSLAB**
## Change Summary

Restructured the eos_cli_config_gen molecule host vars part-4

## Related Issue(s)

Fixes #https://github.com/aristanetworks/avd-internal/issues/171

## Component(s) name

`arista.avd.eos_cli_config_gen`

## Proposed changes
Restructured the eos_cli_config_gen molecule host vars

## How to test
Run eos_cli_config_gen molecule then run tox command to generate the coverage report.

molecule converge -s eos_cli_config_gen
tox -e coverage,report

### User Checklist

- N/A

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly.
- [x] I have updated [molecule CI](https://github.com/aristanetworks/avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

File: ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/host1/ipv6-static-routes.yml [hunk 1]
```diff
@@ -1,42 +1,43 @@
+---
 #### ipv6 static routes ####
 
 ipv6_static_routes:
   - vrf: default
     destination_address_prefix: 2a01:cb04:4e6:d300::/64
-    interface: vlan101
+    interface: vlan1001
```
Comment (MaheshGSLAB) [inline ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/host1/ipv6-static-routes.yml:7]: made the changes as per available vlans and vrfs

File: ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/host1/lacp.yml [hunk 2]
```diff
@@ -1,8 +1,8 @@
+---
 lacp:
   port_id:
     range:
       begin: 1
       end: 128
   rate_limit:
     default: false
-  system_priority: 0
```
Comment (MaheshGSLAB) [inline ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/host1/lacp.yml:8]: LACP system-priority configuration is not allowed if Mlag State is Active( primary/secondary )

File: ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/host1/policy-maps.yml [hunk 3]
```diff
@@ -67,4 +68,4 @@ policy_maps:
         shape: 2000
         bandwidth: 2000
         rate_unit: pps
-      - name: copp-system-aaa
```
Comment (MaheshGSLAB) [inline ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/host1/policy-maps.yml:70]: % Only pre-defined static class maps can be attached to this policy

File: ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/host1/vmtracer-sessions.yml [hunk 4]
```diff
@@ -1,14 +1,15 @@
+---
 ### vmtracer sessions ###
 
 vmtracer_sessions:
   - name: session_1
     url: "https://192.168.0.10"
     username: user1
-    password: "encrypted_password"
+    password: 0011D0516421B120A25735E080A16001D1617
```
Comment (MaheshGSLAB) [inline ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/host1/vmtracer-sessions.yml:8]: default password required as type-7

File: ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/host1/roles.yml [hunk 5]
```diff
@@ -1,4 +1,3 @@
----
```
Comment (alexeygorbunov) [inline ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/host1/roles.yml]: Why are we deleting it?

**Review by alexeygorbunov — APPROVED** [review]
LGTM

File: ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/host1/lacp.yml (hunk 2 above)
Comment (ClausHolbechArista) [reply ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/host1/lacp.yml:8]: This should not stop us from testing it. The config is valid, but you are not allowed to _change it_ if MLAG is already up.

**Review by ClausHolbechArista — APPROVED** [review]

File: ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/host1/roles.yml (hunk 5 above)
Comment (MaheshGSLAB) [reply ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/host1/roles.yml]: fixed thanks

File: ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/host1/lacp.yml (hunk 2 above)
Comment (MaheshGSLAB) [reply ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/host1/lacp.yml:8]: Yes so MLAG config is also there in host1.cfg file so it give issue while apply though CVP hence i removed priority from here and added into host2

**Pull request merged**